docs/api/wallet-rpc.yaml

```
@@ -754,6 +764,22 @@ components:
       items:
         type: string
         example: "2021/10/26 16:40:21,133986791,1,200000000,2680,2680,0.08,"
+    GetinfoResponse:
+      type: object
+      required:
+        - version
+      properties:
+        version:
+          type: string
+          example: "v0.9.10"
```

`JM_CORE_VERSION` is without `v` prefix.
```suggestion
          example: "0.9.10"
```

---

Fixed in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/1532/commits/c9f6ac861033f06ef048d10b46463e15bf2c7480